docs/csharp/programming-guide/nullable-types/handling-null-and-operators.md

```diff
@@ -0,0 +1,159 @@
+# Null coalescing and operators
+
+## Introduction
+Since version 6.0 C# supports the null coalescing and null operator. With these shorthands it is easier to write null checks and thus make your code more readable. On this page is shown how to use these shorthand notations.
```

Comment:
"null operator" should be "null conditional operator"

Comma after "Since version 6.0"

2nd sentence, try this: "These shorthands make it easier to ..."
I'd also change passive voice to active voice.  You say "On this page is shown how to use the shorthand notations." Try "This page shows how to use these shorthand notations". (There are several examples. I'll just say "prefer active voice" in later comments.